release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1201,6 +1201,132 @@ For more information about the unsupported, community-maintained, version of the
 [id="ocp-release-note-cloud-compute-bug-fixes_{context}"]
 ==== Cloud Compute
 
+* When upgrading {gcp-short} clusters that use a boot disk that is not compatible with UEFI, you cannot enable shielded VM support. 
+Previously, this prevented the creation of new machines. 
+With this release, disks with known UEFI incompatiblity have shielded VM support disabled. 
+This primarily affects customers upgrading from {product-title} version 4.12 to 4.13 using the {gcp-short} marketplace images. 
+(link:https://issues.redhat.com/browse/OCPBUGS-17079[OCPBUGS-17079])
+
+* Previously, virtual machines (VMs) in a cluster that ran on {azure-short} failed because the attached network interface controller (NIC) was in a `ProvisioningFailed` state. 
```

Comment:
```suggestion
* Previously, VMs in a cluster that ran on {azure-short} failed because the attached network interface controller (NIC) was in a `ProvisioningFailed` state. 
```
We didn't define it in the preceding note; VM is _probably_ defined way above in the release notes by now. Either I'd define it in the first bullet of this section, or not at all as assume already used.